tests/unit/test_csp_cache.py

```diff
@@ -180,14 +180,14 @@ def test_cache_meter_record(cba_pm, cba_config):
     # usage record
     cache_meter_record(
         cache=cache,
-        record_id=test_record_id,
+        record_ids=test_record_id,
```

Comment:
As @KeithMnemonic pointed out, the test_record_id value, as assigned on line 153, is a string value, but it should be a list. And the variable name should be updated to reflect the plurality change.